src/app/domains/Interview/components/lists/InterviewSimpleList/InterviewSimpleList.style.js

```diff
@@ -0,0 +1,7 @@
+export const dots = {
+  display: 'inline-block',
+  width: '100%',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis'
+}
```

Comment:
Maybe instead of ellipsis use [word-break](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) property witch antd-styled `Text` component already exist